translations/kubectl/ja_JP/LC_MESSAGES/k8s.po

```diff
@@ -0,0 +1,92 @@
+# Test translations for unit tests.
+# Copyright (C) 2017
+# This file is distributed under the same license as the PACKAGE package.
+# FIRST AUTHOR girikuncoro@gmail.com, 2017.
+#
+msgid ""
+msgstr ""
+"Project-Id-Version: gettext-go-examples-hello\n"
+"Report-Msgid-Bugs-To: \n"
+"POT-Creation-Date: 2013-12-12 20:03+0000\n"
+"PO-Revision-Date: 2017-01-29 22:54-0800\n"
+"Last-Translator: Giri Kuncoro <girikuncoro@gmail.com>\n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=UTF-8\n"
+"Content-Transfer-Encoding: 8bit\n"
+"X-Generator: Poedit 1.6.10\n"
+"X-Poedit-SourceCharset: UTF-8\n"
+"Language-Team: \n"
+"Plural-Forms: nplurals=2; plural=(n > 1);\n"
+"Language: ja\n"
+
+# https://github.com/kubernetes/kubernetes/blob/masterpkg/kubectl/cmd/apply.go#L98
```

Comment:
Seems those url all pointing to the wrong line. Does that matters ? also I saw there's a `template.pot` file. Does this file auto-generate content when command changed ?  @brendandburns